src/logging.cc

```diff
@@ -1306,12 +1306,17 @@ void LogCleaner::Run(const std::chrono::system_clock::time_point& current_time,
   assert(enabled_);
   assert(!base_filename_selected || !base_filename.empty());
 
+  if (next_cleanup_time_map_.find(base_filename) == next_cleanup_time_map_.end()) {
+    next_cleanup_time_map_[base_filename] = std::chrono::system_clock::time_point{};
+  }
+  auto next_cleanup_time = next_cleanup_time_map_.at(base_filename);
```

Comment:
`at` can [throw `std::out_of_range`](https://en.cppreference.com/w/cpp/container/unordered_map/at) that is not caught anywhere.